wdl-lint/src/rules/runtime_section_keys.rs

```diff
@@ -342,6 +342,14 @@ impl Rule for RuntimeSectionKeysRule {
             SyntaxKind::RuntimeSectionNode,
         ])
     }
+
+    fn related_rules(&self) -> &[&'static str] {
+        &[
+            "DeprecatedObject",
+            "DeprecatedPlaceholderOption",
+            "MissingRequirements",
```

Comment:
```
         For WDL v1.2 documents and later, this rule does not evaluate because `runtime` sections \
         were deprecated in this version."
```
so remove as `MissingRequirements` only applies to WDL v1.2